app/src/forms/email/emailService.js

```diff
@@ -1,7 +1,8 @@
-const chesService = require('../../components/chesService');
 const fs = require('fs');
-const log = require('npmlog');
 const path = require('path');
+
```

Comment:
Remove whitespace

Reply:
This whitespace line is intended as it separates imports from node_modules vs imports from our own codebase.